app/build.gradle.kts

```diff
@@ -179,18 +179,18 @@ dependencies {
     implementation("androidx.biometric:biometric:1.1.0")
     implementation("androidx.webkit:webkit:1.6.0")
 
-    implementation("com.google.android.exoplayer:exoplayer-core:2.18.2")
-    implementation("com.google.android.exoplayer:exoplayer-hls:2.18.2")
-    implementation("com.google.android.exoplayer:exoplayer-ui:2.18.2")
-    "fullImplementation"("com.google.android.exoplayer:extension-cronet:2.18.2")
-    "minimalImplementation"("com.google.android.exoplayer:extension-cronet:2.18.2") {
+    implementation("com.google.android.exoplayer:exoplayer-core:2.18.3")
+    implementation("com.google.android.exoplayer:exoplayer-hls:2.18.3")
+    implementation("com.google.android.exoplayer:exoplayer-ui:2.18.3")
+    "fullImplementation"("com.google.android.exoplayer:extension-cronet:2.18.3")
+    "minimalImplementation"("com.google.android.exoplayer:extension-cronet:2.18.3") {
         exclude(group = "com.google.android.gms", module = "play-services-cronet")
     }
     "minimalImplementation"("org.chromium.net:cronet-embedded:108.5359.79")
 
     implementation(platform("androidx.compose:compose-bom:2023.01.00"))
     implementation("androidx.compose.animation:animation")
-    implementation("androidx.compose.compiler:compiler:1.4.2")
+    implementation("androidx.compose.compiler:compiler:1.4.3")
```

Comment:
Don't you also need to bump the compiler version here: https://github.com/home-assistant/android/blob/adf646b24d2483d7ddeb44946c25cc0537ddfe35/app/build.gradle.kts#L47

Reply:
I probably should - interesting it didn't complain during build

Reply:
> I probably should - interesting it didn't complain during build

it did for me previously but that was due to compose vs kotlin vesrion mismatch. It probably only cares if it cannot compile correctly due to minimum version requirements?